scheduler/rank.go

```diff
@@ -206,10 +206,35 @@ OUTER:
 			continue
 		}
 
-		// Index the existing network usage
+		// Index the existing network usage.
+		// This should never collide, sice it represents the current state of
```

Comment:
```suggestion
		// This should never collide, since it represents the current state of
```

I know you were just testing me :grin:

Reply:
Passed!